fhir-audit/src/main/java/com/ibm/fhir/audit/logging/impl/WhcAuditCadfLogService.java

```diff
@@ -82,7 +82,7 @@
     private static CadfResource observerRsrc =
             new CadfResource.Builder("fhir-server", ResourceType.compute_node)
                             .geolocation(new CadfGeolocation.Builder(geoCity, geoState, geoCountry, null).build())
-                            .name("Fhir Audit")
+                            .name("IBM FHIR Server - Audit")
                             .host(System.getenv("HOSTNAME"))
```

Comment:
I know it's not part of the change, but is this the best way to get the name of the host? Do we ensure that HOSTNAME is part of the environment?  On Windows, COMPUTERNAME might be more appropriate. Let me know if you think we need a ticket for this.

Reply:
Good point, I don't think it's super reliable. I think a ticket isn't bad, probably lower priority.

Reply:
Thanks